src/foundation_models/SoftCON/models/dinov2/vision_transformer.py

```diff
@@ -55,6 +55,7 @@ def forward(self, x):
         return x
 
 
+# TODO is this needed? Or can also be loaded from the HUB?
```

Comment:
@wangyi111is all of this necessary, and couldn't this just be loaded from torch.hub similar to how the dinov2 wrapper works? Then we could remove this

Reply:
okay, removed now in the latest commit, that should make things easier.